### ISSUE TITLE
Arrow Key Navigation Fails Inside Ctrl+E Modal

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1660]
When the Ctrl+E modal is open, the arrow keys work outside of the modal but not within it.

Steps to reproduce:
1. Open the drive
2. Open the Ctrl+E modal
4.  Use the arrow keys (you'll see that they work outside the modal)
